### DISCUSSION TITLE
Itwin.js - Civil Design Review Tool, Profile reviewer

### DISCUSSION THREAD

**Discussion by andrewwrichardson** [iTwin/itwinjs-core#3864] [General]
Hi All,

Is it possible to load the profile tool from the civil design review tools into an app developed using Itwin.js.

I noticed their was some discussions about this previously nearly a year ago.

https://github.com/iTwin/itwinjs-core/discussions/2264

**Comment by pmconne** [comment]
@aruniverse do you know of any progress in this area?

**Reply by aruniverse** [reply]
Unfortunately at the moment, it's still not possible to add those tools to your own iTwin.js application, and you'll have to continue to use Design Review for those features. But we do still hope to make those publicly available later. I'll update this thread once we do.
